docs/en/signal-hub-endpoint.rst

```diff
@@ -0,0 +1,169 @@
+.. include:: ../common/common_definitions.rst
+
+.. "included" file, so we start with '-' title level
+
+.. role:: raw-html(raw)
+  :format: html
+
+
+PDND Signal Hub Endpoints
+-------------------------
+Within the PDND platform, the Signal Hub acts as an intermediary between a PDND Provider and its PDND Consumers to signal data variations. To achieve that, the manager of the PDND platform, hereafter the PDND Manager, provides (playing the role of PDND e-Service Provider) two PDND Signal Hub e-Services:
+  - the Signal Collection e-Service which is used by PDND e-Service Providers to deposit Signals (here, the PDND e-Service Provider acts as Consumer of the Signal Collection e-Service);
+  - the Signal Distribution e-Service which is used by PDND e-Service Consumers to retrieve collected Signals (here, the PDND e-Service Consumer also acts as Consumer of the Signal Distribution e-Service).
```

Comment:
```suggestion
  - the Signal Distribution e-Service which is used by PDND e-Service Consumers to retrieve collected Signals, since the PDND e-Service Consumer also acts as Consumer of the Signal Distribution e-Service.
```